game-core/src/main/java/games/strategy/triplea/delegate/power/calculator/PowerStrengthAndRolls.java

```diff
@@ -124,4 +124,15 @@ public int getStrength(final Unit unit) {
   public int getRolls(final Unit unit) {
     return totalStrengthAndTotalRollsByUnit.get(unit).getRolls();
   }
+
+  @Override
+  public int calculatePower(final Unit unit) {
+    return totalStrengthAndTotalRollsByUnit.get(unit).calculatePower();
+  }
+
+  @Override
+  public TotalPowerAndTotalRolls buildOpposite() {
+    return PowerStrengthAndRolls.build(
+        totalStrengthAndTotalRollsByUnit.keySet(), calculator.buildOppositeCombatValue());
```

Comment:
Since the unit is ordered differently, this new PowerStrengthAndRolls will not be as optimized.  But the only use of `buildOpposite` is when the combat value has no unit support so it won't actually be affected.  Should I add a comment about that?  Or maybe change this method to always create a non-supported combat value?